File: JMS.DAL/Models/AssessmentQuestion.cs
using JMS.DAL.Common.Enums;
using System;
using System.Collections.Generic;

namespace JMS.DAL.Models
{
    public partial class AssessmentQuestion
    {
        public AssessmentQuestion()
        {
            AssessmentResult = new HashSet<AssessmentResult>();
        }

        public int Id { get; set; }
        public string Question { get; set; }
        public QuestionCategory Category { get; set; }
        public int? Weight { get; set; }
        public bool IsThirdParty { get; set; }
        public int? CheckpointId { get; set; }
        public int? JourneyId { get; set; }

        public virtual Checkpoint Checkpoint { get; set; }
        public virtual Journey Journey { get; set; }

        public virtual IEnumerable<AssessmentResult> AssessmentResult { get; set; }
    }
}

File: JMS.WEB/GMSSolution/src/app/shared/Entities/User/assessment-question.ts
import { AssessmentResult } from './assessment-result';
import { QuestionCategory } from './question-category.enum';
import { Journey } from './journey';
import { Checkpoint } from './checkpoint';

export class AssessmentQuestion {
  Id: Number;
  Question:string;
  Weight:number;
  CheckpointId:number;
  JourneyId:number;
  IsThirdParty:boolean;
  Checkpoint:Checkpoint;
  Category:QuestionCategory;
  Journey:Journey;
  AssessmentResult:AssessmentResult[];

}
